drivers/pds/lib/utils.go

```diff
@@ -1388,7 +1388,7 @@ func CreateIndependentMySqlApp(ns string, podName string, appImage string, pvcNa
 	return pod, podName, nil
 }
 
-//CreatePodWorkloads generate workloads as standalone pods
+// CreatePodWorkloads generate workloads as standalone pods
```

Comment:
I believe this is added when you run Makefile.

Reply:
Same here